src/loaders/compute_tools/microtrait/env_microtrait.yaml

```diff
@@ -39,4 +39,4 @@ dependencies:
   - pandas=2.0.0
   - unzip=6.0
   - jsonlines=3.1.0
-  - pydantic=1.10.12
```

Comment:
`import heatmap_common_models` with pydantic 1 does break.

Reply:
That's surprising. What's the error?

Reply:
ah oh. cleared the terminal. But I recall trackback is from `heatmap_common_models`.

Reply:
Weird, I can't understand how those changes could make pydantic 1 puke